english_tasks/irregular_verbs.py

```diff
@@ -32,36 +52,36 @@ def random_verb(list_of_verbs):
     :param list_of_verbs:
     :return:
     """
-    random_index = random.randint(0, len(list_of_verbs) - 1)
+    random_index = secrets.randbelow(len(list_of_verbs) - 1)
```

Comment:
return back to using `random` module

Reply:
I think the secret module is better than random module because if you use random, you might find the smaller range of verbs than secret module has.

Reply:
I think you don't know what difference will it make for the user